ynput/OpenPype
# Fusion: Resolution validator

## Changelog Description
Added a resolution validator.
The code is from my old PR (https://github.com/ynput/OpenPype/pull/4921) that I closed because the PR also contained a frame range validator that no longer is needed.

## Additional info
The validator is optional as you might want/need to render out different resolutions than what's set for the asset.

## Testing notes:
1. Make a BG node with a different resolution than your asset
2. Create a new saver and pipe in the BG
3. Validate and see an error pop up
